doc/tech/README.md

```diff
@@ -0,0 +1,148 @@
+[<< Back](https://cntt-n.github.io/CNTT/)
+# Common NFVI Telco Taskforce
+
+## Table of Contents
+* [1. Overview](#1.0)
+* [2. Scope](#2.0)
+* [3. Principles](#3.0)
+* [4. Technology Direction](#4.0)
+  * [4.1 IO Virtualisation](#4.1)
+  * [4.2 SmartNICs](#4.2)
+  * [4.3 Acceleration Cards](#4.3)
+  * [4.4 FPGA](#4.4)
+  * [4.5 GPUs/NPUs](#4.5)
+  * [4.6 EPA/NFD](#4.6)
+* [5. Available Specifications](#5.0)
+
+<a name="1.0"></a>
+## 1. Overview
+
+Initially organized early in 2019, the Common Network Function Virtualisation Infrastructure Telecom Taskforce (CNTT) was initially created in response to rapid changes in how networking applications are being designed, built and managed, plus a growing recognition of a perceived functional gap between the previous standard infrastructure models and the architectures needed to support Network Function Virtualization (NFV) applications.  Organizationally the Common Telco Network Function Virtualisation Infrastructure (NFVI)  project, jointly hosted by GSMA and the Linux Foundation, operates as an open committee responsible for creating and documenting an industry aligned Common NFVI Framework.  The CNTT group was created with the intent that it would create the NFVI framework, and eventually morph into an on-going project under the auspices of the GSMA and the Linux Foundation umbrellas.  The final on-going operational form of the group will be determined as the project evolves. 
+
+<a name="2.0"></a>
+## 2. Scope
+
+Within the framework of the Common Telecom NFVI vision, there are three levels of documents needed to document the components and allow the practical application of the systems. They are, as highlighted in **Figure 1**:  **Reference Model**, **Reference Architecture**, **Reference Implementation**, and **Reference Certification**.
```

Comment:
IMHO, the scope is not the artefacts produced -- it is 

The CNTT aims to establish an open-sourced end-to-end cloud infrastructure ecosystem for certification of network functions (VNF/CNF). Towards that end, a number of artefacts specifying the cloud infrastructure need to be produced (Figure 1). These are  organised in three layers: the technology agnostic Reference Model (RM), the technology/orchestrator specific Reference Architectures (RA), the Reference Implementation (RI) for a specific RA with the bill of materials.

Reply:
@pgoyal01 if it is OK, since this PR aim is not proposing text change (it is a simple move), I propose to keep those sections not changed and we can have a different PR to propose any text change.